march_hardware/src/IMotionCube.cpp

```diff
@@ -296,6 +301,11 @@ float IMotionCube::getIMCVoltage()
          static_cast<float>(imc_voltage_iu);  // Conversion to Volt, see Technosoft CoE programming manual
 }
 
+float IMotionCube::getMotorVoltage()
+{
+  return this->read16(this->miso_byte_offsets_.at(IMCObjectName::DCLinkVoltage)).ui;
```

Comment:
DCLinkVoltage is the other voltage or am I wrong? At least this is the same object as used in the `getIMCVoltage()`

Reply:
That was a mistake with copying the lines, after fixing all the merge conflicts :(